docs/index.rst

```diff
@@ -3,49 +3,60 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to tdastro's documentation!
+TDAstro
 ========================================================================================
 
-Dev Guide - Getting Started
----------------------------
+Time-Domain Forward-Modeling for the Rubin Era
+-------------------------------------------------------------------------------
 
-Before installing any dependencies or writing code, it's a great idea to create a
-virtual environment. LINCC-Frameworks engineers primarily use `conda` to manage virtual
-environments. If you have conda installed locally, you can run the following to
-create and activate a new environment.
+Realistic light curve simulations are essential to many time-domain problems. 
+simulations are needed to evaluate observing strategy, characterize biases, 
+and test pipelines.The need for a flexible, scalable, and user-friendly time-domain
+simulation software has increased as the new survey telescopes get ready for their
+first lights. TDAstro aims to provide such software for the time domain community.
 
-.. code-block:: console
+.. figure:: _static/tdastro-intro.png
+   :class: no-scaled-link
+   :scale: 80 %
+   :align: center
+   :alt: TDAstro simulation components
 
-   >> conda create env -n <env_name> python=3.10
-   >> conda activate <env_name>
+   TDAstro simulation components
 
+The main simulation components in TDAstro include:
 
-Once you have created a new environment, you can install this project for local
-development using the following commands:
+* ``PhysicalModel`` defines the properties of the time-domain source, which can 
+  also include a host-galaxy model.
+* ``EffectModel`` describes how the SED changes (e.g. redshifting, lensing, dust)
```

Comment:
We took out effect models. They are no incorporated into the simulation code directly.